plugins/org.yakindu.sct.doc.user/src/user-guide/overview.textile

```diff
@@ -3,89 +3,98 @@ p.
+We extend the switch example above into a light switch with different brightness levels. The light switch has two buttons, an _ON_ button and an _OFF_ button. Pressing the _ON_ button switches the light on and toggles through the different brightness levels. This behavior is modeled by the state machine below. Pressing a button raises a corresponding event (_ON_pressed_ or _OFF_pressed_) upon which the machine reacts with a state change and corresponding outputs/actions. The output of the state machine is simply the brightness value. As in Moore machines only states produce outputs, we need one dedicated state per brightness level:
 
-h2(#overview_mealy). Compact state machines by George H. Mealy
+!(small)images/overview_moore.png(Light switch example as a Moore machine, modeled with YAKINDU Statechart Tools)!
 
-In comparison with the Moore machines seen above, "_Mealy_ machines":https://en.wikipedia.org/wiki/Mealy_machine produce outputs only on transitions and not in states. This often results in state diagrams with fewer states because more logic can be put on transitions. As a Mealy machine, the ferry example looks like the following image:
+p=. Light switch example as a Moore machine, modeled with YAKINDU Statechart Tools
 
-!(large)images/ferry_mealy_1.png(Ferry example as a Mealy machine)!
 
-p=. Ferry example as a Mealy machine
+*Mealy machines*
 
-States do not produce outputs anymore, but the transitions do. Whether the ferry moves northwards or southwards is now expressed by the corresponding transitions. The input event mate.sail may lead to the output _ferry.moveSouthwards_ or _ferry.moveNorthwards_ depending on the active state. The state machine offers an extended logic: the states *Northwards* and *Southwards* have two transitions that lead to different states depending on the input event: If the event _mate.sail_ is raised in *Northwards* the output is _ferry.moveSouthwards_, if _quay.reachedNorthernBank_ is raised the ferry stops. The same logic as a Moore machine would lead to additional states like the following image shows:
+In comparison with Moore machines, "_Mealy_ machines":https://en.wikipedia.org/wiki/Mealy_machine *produce outputs only on transitions* and not in states. This often results in state diagrams with fewer states because more logic can be put on transitions. 
 
-!(large)images/ferry_moore2.png(The ferry as an extended Moore machine)!
+In the light switch example we can now merge all _Light On_ states into one and produce the output on a transition. Here we just increment the brightness value each time the transition is taken. We use the modulo expression to ensure the brightness value stays between 1 and 3.
```

Comment:
This describes no Mealy FSM. FSM have a fixed output alphabet and do not know the concept of variables.